public/js/index.js

```diff
@@ -1,5 +1,31 @@
-function displayMessage() {
-  return 'Hello, World!';
+document.body.appendChild(buildProducts(products));
+
+function buildProducts(products) {
+  var div = document.createElement("div");
```

Comment:
We are integrating a linter in a separate issue https://github.com/DanyDin1801/Product-Portal/pull/19 

I think we should care about the code style later because the linter will reveal all the style issues automatically.